openedx_unidigital/handlers.py

```diff
@@ -117,7 +118,15 @@ def add_user_to_team(user, team_id: str) -> None:
             team.add_user(user)
             log.info(f"The user='{user}' has been added to the team='{team}'.")
         except AlreadyOnTeamInTeamset:
-            log.exception(f"The user='{user}' is already on a team in the teamset.")
+            old_membership = CourseTeamMembership.objects.filter(
+                user=user,
+                team__course_id=team.course_id,
+                team__topic_id=team.topic_id,
+            ).first()
+            old_membership.delete()
+            log.info(
+                f"The user='{user}' was moved from the team='{old_membership.team} to the team='{team}'."
+            )
```

Comment:
This could be at a debug level so it doesn't add more noise to the logs. What do you say?

Reply:
Yes, I agree